Tools/autotest/common.py

```diff
@@ -560,7 +560,7 @@ def arm_motors_with_rc_input(self, timeout=20):
                 self.set_output_to_trim(self.get_rudder_channel())
                 self.progress("Arm in %ss" % arm_delay)  # TODO check arming time
                 return True
-        self.progress("FAILED TO ARM WITH RADIO")
+        self.progress("Unable to ARM with radio")
```

Comment:
The original message was closer to the truth.  "Unable to" means it wasn't possible; "FAILED TO" means you attempted to and it didn't work.  Subtle but important difference.

Same applies to similar changes here.

Reply:
arm_vehicle method is also using "unable". I don't want to use FAILED as we got test that need this to fail explicitly... Looking at failure on travis log is as simple as ctrl+f FAIL . If we keep this message we will also see the one that are wanted ...
I am ok to change the wording for something than fail !

Reply:
Use mixed-case `Failed to disarm with radio`